packages/create-ship-app/src/index.js

```diff
@@ -18,13 +18,14 @@ let deploymentType;
   console.log(`Hey! Let’s build your ${gradient.pastel('Ship')} 🚀`);
   
   const args = utils.getCLIArgs();
-  
-  if (args[0] === 'init') {
+  const [firstArg] = args;
+  if (firstArg === 'init') {
     projectName = await buildSteps.askProjectName();
   } else {
-    projectName = args[0];
-    console.log(`Project name: ${projectName}`);
+    projectName = firstArg;
   }
+
+  console.log(`Project name: ${projectName}`);
```

Comment:
let's return this in else block

`askProjectName` already has message about project name `What’s your project name: `